edit-post/components/meta-boxes/meta-boxes-area/index.js

```diff
@@ -86,4 +86,9 @@ function mapStateToProps( state ) {
 	};
 }
 
-export default connect( mapStateToProps )( MetaBoxesArea );
+export default connect(
```

Comment:
With #5137, maybe update to use `withSelect` ? Might require registering more/all selectors for `edit-post` (also done in #5206).

Reply:
I think we should do a separate PR and move all of these at once :)